bin/inference/pycbc_inference

```diff
@@ -70,6 +81,13 @@ parser.add_argument("--samples-file", default=None,
                          "The priors must allow encompass the initial "
                          "positions from the InferenceFile being read.")
 
+# calibration data options
+parser.add_argument("--calib", action="store_true", default=False,
+                  help="Switch to determine whether calibration parameters are"
+                       " included in likelihood calculation.")
+parser.add_argument("--transfer-functions", type=str,
+                  help="Path to hdf file containing calibration data")
```

Comment:
Also, with the configuration file you can do it on the command line as well. So not really a loss of functionality.